eos-firewall-localonly

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash -e
```

Comment:
Nitpick: This would be so much more readable as a Python script. (Not expecting this review comment to get addressed. Just venting some hatred for bash a bit.)